app/routes/account_.$accountId.update/route.tsx

```diff
@@ -141,11 +141,15 @@ export default function UpdateAccount() {
   return state !== "idle" ? (
     <LoadingOverlay
```

Comment:
not necessary, but ive seen this same structure passed enough that it could be abstracted into a component

Reply:
https://github.com/pokt-foundation/portal-platform/pull/600